myriad-scheduler/src/main/java/org/apache/myriad/configuration/MyriadConfiguration.java

```diff
@@ -197,47 +200,44 @@ public String getFrameworkRole() {
     return nmInstances;
   }
 
-  public Boolean isRebalancer() {
-    return rebalancer != null ? rebalancer : DEFAULT_REBALANCER;
+  public Boolean isRebalancerEnabled() {
+    return Optional.of(rebalancerEnabled).or(DEFAULT_REBALANCER_ENABLED);
   }
 
   public Boolean isHAEnabled() {
-    return haEnabled != null ? haEnabled : DEFAULT_HA_ENABLED;
+    return Optional.of(haEnabled).or(DEFAULT_HA_ENABLED);
   }
 
   public NodeManagerConfiguration getNodeManagerConfiguration() {
-    return this.nodemanager;
+    return nodeManager;
   }
 
-  public Map<String, ServiceConfiguration> getServiceConfigurations() {
-    return this.services;
+  public Optional<Map<String, ServiceConfiguration>> getServiceConfigurations() {
+    return Optional.fromNullable(services);
   }
```

Comment:
Seems like just sending an empty Map here would avoid all the null and isPresent checks later.  Thoughts?